.travis.yml

```diff
@@ -7,7 +7,7 @@ before_install:
 - docker push karrot96/todo-app
 - docker build --target test --tag my-test-image --cache-from karrot96/todo-app .
 script:
-- docker run -e USERNAME -e PASSWORD -e MONGO_URL -e DEFAULT_DATABASE my-test-image
+- docker run -e DB_USER -e PASSWORD -e MONGO_URL -e DEFAULT_DATABASE -e WRITER_ID=19879648 my-test-image
```

Comment:
It would be more consistent to set the value of this new variable in your environment variables